browse/controllers/files/__init__.py

```diff
@@ -10,6 +10,8 @@
 
 
 BUFFER_SIZE = 1024 * 4
+CACHE_AGE_SEC_VERSIONED = 60 * 60 * 24 * 7
+CACHE_AGE_SEC_UNVERSIONED = 60 * 60 * 24
```

Comment:
not sure we are too worried about it, but this does mean replacements will consistently not be visible at publish/announce time if someone looked at the non versioned paper the day before.

Reply:
> not sure we are too worried about it, but this does mean replacements will consistently not be visible at publish/announce time if someone looked at the non versioned paper the day before.

It would, so we also purge the previous nights announcements at a few points.
And yes, it would be terrible for us to announce a replacement, and then send users an older pdf!